lib/cqm/converter/bonnie_measure.rb

```diff
@@ -79,7 +79,8 @@ def self.to_cqm(bonnie_measure)
 
           population_map[population_name.to_sym] = CQM::StatementReference.new(
             library_name: cqm_measure.main_cql_library,
-            statement_name: get_cql_statement_for_bonnie_population_key(bonnie_measure.populations_cql_map, population_key)
+            statement_name: get_cql_statement_for_bonnie_population_key(bonnie_measure.populations_cql_map, population_key),
+            hqmf_id: bonnie_measure.population_criteria[population_key]['hqmf_id']
```

Comment:
The `population_key` is from the value side of the this structure in the measure at `populations[n]`:
![image](https://user-images.githubusercontent.com/11613067/54202792-2c00fc00-44a7-11e9-9cb0-c3fa6f3bbf70.png)

The multiple populations are handled since that key is used to reference more details about the populations in the `population_criteria` which looks like this:
![image](https://user-images.githubusercontent.com/11613067/54202995-a92c7100-44a7-11e9-9d48-36a149b2bab7.png)

Reply:
got it.  and that population in is the HDS model.